hardhat/test/ApeProtocol/ApeVault.ts

```diff
@@ -0,0 +1,43 @@
+import { Contract, Signer } from 'ethers';
+import { ethers, deployments } from 'hardhat';
+import { expect } from 'chai';
+
+import {
+  ApeVaultFactory__factory,
+  ApeVaultWrapper__factory,
+} from '../../typechain';
+
+describe('Ape Vault', () => {
+  const USDC_ADDRESS = '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48';
+  const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+  let accounts: Signer[];
+
+  before(async () => {
+    accounts = await ethers.getSigners();
+    await deployments.fixture();
+  });
+
+  it('Test vault creation', async () => {
```

Comment:
please use `test` here instead of `it`

`it` is only meant to be used when, in combination with the first argument, it makes an English sentence. Like
```
it('can be created', async () => {
```

It refers to the subject of the `describe` block, in this case 'Ape Vault'.

Reply:
This is just the draft PR, I was going to improve docs afterwards.